plugins/woocommerce/tests/e2e-pw/tests/customize-store/intro.spec.js

```diff
@@ -0,0 +1,179 @@
+const { test, expect, request } = require('@playwright/test');
+const { BASE_URL } = process.env;
+const { features } = require('../../utils');
+const { activateTheme } = require('../../utils/themes');
+const { setOption } = require('../../utils/options');
+
+const CUSTOMIZE_STORE_URL =
+	'/wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store';
+
+const skipTestIfUndefined = () => {
+	const skipMessage = `Skipping this test on daily run. Environment not compatible.`;
+
+	test.skip(() => {
+		const shouldSkip = BASE_URL !== undefined;
+
+		if (shouldSkip) {
+			console.log(skipMessage);
+		}
+
+		return shouldSkip;
+	}, skipMessage);
+};
+
+skipTestIfUndefined();
```

Comment:
I'm not sure that this is still necessary - I didn't add it https://github.com/woocommerce/woocommerce/pull/45804